include/api.php

```diff
@@ -953,7 +953,7 @@ function api_account_verify_credentials($type)
 	// - Adding last status
 	if (!$skip_status) {
 		$user_info["status"] = api_status_show("raw");
-		if (!count($user_info["status"])) {
+		if (!is_array($user_info["status"]) || !count($user_info["status"])) {
```

Comment:
`api_status_show('raw')` can return void if there aren't any last public message. You'd avoid an issue with the count but you'd get an error with `unset($user_info["status"]["user"]);` on line 959.

Reply:
I see. That's a code smell too I think, because we're mixing return and not returning in one function.